CMakeLists.txt

```diff
@@ -247,6 +248,12 @@ if(HTTPS_NOT_AVAILABLE)
 	add_definitions(-DHTTPS_NOT_AVAILABLE)
 endif()
 
+# Disable the usage of MMAP for the memory system.
+# It is not tested on all platforms and can cause issues.
+if(USE_NO_MMAP)
+	add_definitions(-DNO_MMAP)
```

Comment:
gonna want to combine it with -DMASKED_PSP_MEMORY I think..

Reply:
Do you want me to add both flags then? Should we then use a different key name?

Reply:
You can just add both flags here.

Reply:
btw does mmap not working on Alpine? because the unittest works fine when i tested it on alpine chroot at https://github.com/ANR2MERefork/ppsspp/actions/runs/10104071293
Headless test also worked, at least for x86/x64 arch.

Reply:
It does work on Alpine, and this is why as you said it passes tests, however in order to work (inside of the docker) we have needed to set the SHM-MEM value.
This is not ideal, as every single user wants to use PPSSPPHeadless from a Docker, they need to set this manually, said that for our use case (testing PSPSDK binaries) is enough.

Reply:
> You can just add both flags here, in the same add_definitions

Done